### PR TITLE
1.8.5 CHANGELOG

### PR DESCRIPTION
**Pull request by rturner3**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+## [1.8.5] - 2023-11-22
+
+### Added
+
+- All credential types supported by Azure can now be used in `azure_msi` NodeAttestor plugin and `azure_key_vault` KeyManager plugin (#4568)
+- `EnableHostnameLabel` field in Server and Agent `telemetry` configuration section that enables addition of a hostname label to metrics (#4584)
+
+### Changed
+
+- Agent SDS API now provides a SPIFFEValidationContext as the default CertificateValidationContext (#4618)
```
Comment (azdagron) [inline CHANGELOG.md:12]: This was already the case for Envoy 1.18+, we just made it the default if we couldn't detect the Envoy version (inverted the check).... not sure how we want to word it, maybe:
```suggestion
- Agent SDS API now provides a SPIFFEValidationContext as the default CertificateValidationContext when the Envoy version cannot be determined (#4618)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+- Server CAs now contain a `serialNumber` DN in the `Subject` field (#4585)
```
Comment (azdagron) [inline CHANGELOG.md:13]:
```suggestion
- Server CAs now contain a `serialNumber` attribute in the `Subject` DN (#4585)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+- Improved accuracy of Agent log message for SVID renewal events (#4654)
+
+### Deprecated
+
+- `use_msi` configuration fields in `azure_msi` NodeAttestor plugin and `azure_key_vault` KeyManager plugin are deprecated in favor of the chained Azure SDK credential loading strategy (#4568)
+
+### Fixed
+
+- Agent SDS API now provides correct CertificateValidationContext when Envoy is not initially registered in SPIRE then later registered (#4611)
```
Comment (amartinezfayo) [inline CHANGELOG.md:22]: I'm not sure if this sentence is properly formulated?

**Review by amartinezfayo — COMMENTED** [review]
Looks great!

File: CHANGELOG.md (hunk 3 above)
Comment (rturner3) [reply CHANGELOG.md:22]: Any suggested wording tweaks? Let me know if something is not coming across clearly here

Comment (rturner3) [reply CHANGELOG.md]: Reworded this a bit to try to improve the flow

**Review by amartinezfayo — APPROVED** [review]
LGTM, thank you @rturner3!

**Review by azdagron — APPROVED** [review]

**Pull request merged**